release_notes/topics/15_1_0.adoc

```diff
@@ -0,0 +1,39 @@
+= Highlights
+
+== Quarkus distribution
+
+Without comparison the biggest highlight of this release is all the improvements that have been made to the Quarkus distribution. So, many in fact that it will be hard to list them all.
+
+The CLI has been polished to hell and back, and we believe it now provides a very simple and convenient approach to configuring and running Keycloak. It's almost so simple that documentation shouldn't be needed.
+
+That doesn't mean we don't plan to provide documentation for configuring Keycloak, but it didn't quite make the quite this time around. In lack of documentation expect a blog post to follow the release introducing all the changes to the Quarkus distribution, as well as an overview on how to use it.
```

Comment:
Typo: "quite" doubled in first sentence. "..., but it didn't quite make it this time around."